interface/code_systems/dataloads_ajax.php

```diff
@@ -36,7 +36,7 @@
 <html>
 <head>
 <title><?php echo xlt('External Data Loads'); ?></title>
-<?php Header::setupHeader(['jquery-ui', 'jquery-ui-lightness']); ?>
+<?php Header::setupHeader(); ?>
```

Comment:
breaks script, since this gui appears very reliant on jquery-ui accordion:

![image](https://user-images.githubusercontent.com/278968/75851087-a6541b80-5d9d-11ea-9cba-1b974404ff44.png)

Reply:
This script looks like a potentially big issue. Let me know if you want me to break this script out into a separate issue.

Reply:
Splitting sounds great 👀

Reply:
Placed a separate issue for this here: https://github.com/openemr/openemr/issues/3077
(so, you can revert the change in this script and move along :) )